README.md

```diff
@@ -120,7 +121,6 @@ That's it! Congratulations, you have created your database and connected to it.
 
 ## 🔭 Future Features <a name="future-features"></a>
 
-- Add _**join table**_ for visits.
 - Database performance audit.
 - Add database schema diagram.
 
```

Comment:
- [ ] OPTIONAL
MIT license is broken on _line 151_ of your ReadMe, kindly endeavor to fix it.